Scripts/altapay_config.php

```diff
@@ -273,6 +273,24 @@ public function launch()
             }
         }
 
+        $taxRate = $this->_objectManager->create(\Magento\Tax\Model\Calculation\Rate::class);
+        $taxRate->loadByCode('DKTaxRate');
+
+        // Create tax rate if not exists
+        if (!$taxRate->getId()) {
+            $taxRate = $this->altapayCreateTaxRate($taxRate);
+        }
+
+        $taxRateId = $taxRate->getId();
+
+        $taxRule = $this->_objectManager->create(\Magento\Tax\Model\Calculation\Rule::class);
+        $taxRule->load('AltaPayTaxRule', 'code');
+
+        // Create tax rule and assign tax rate
+        if (!$taxRule->getId()) {
+            $this->altapayCreateTaxRule($taxRule, $taxRateId);
+        }
+
```

Comment:
This seems to be affecting the main configuration file, shouldn't this be documented in the README as a change?

Reply:
@emicha this is the part of integration tests, so no release notes for this. We can safely merge this pull request.